doc/Development_Documentation/07_Workflow_Management/05_Workflow_Tutorial.md

```diff
@@ -233,10 +233,12 @@ And, the transition with a *"timeWorked"* field.
         publish:
             from: content_prepared
             to: accepted
-            guard: "is_fully_authenticated() and has_role('ROLE_PIMCORE_SUPERUSER')"
+            guard: "is_fully_authenticated() and is_granted('ROLE_PIMCORE_SUPERUSER')"
```

Comment:
is that change really necessary?

Reply:
I think it should be something like `guard: "is_fully_authenticated() and 'ROLE_PIMCORE_SUPERUSER' in role_names"` ?

Reply:
`is_granted` is the official replacement of `has_role` therefore I think this is correct in this case: https://symfony.com/blog/symfony-4-2-curated-new-features#security (second bulletpoint)